STPopupPreview/STPopupPreviewRecognizer.m

```diff
@@ -316,11 +316,19 @@ - (void)gestureAction:(UIGestureRecognizer *)gesture
             _popupController.containerView.layer.cornerRadius = 10;
             _popupController.transitionStyle = STPopupTransitionStyleFade;
             _popupController.hidesCloseButton = YES;
-            if (NSClassFromString(@"UIVisualEffectView")) {
+            
+            if([NSArray respondsToSelector:@selector(arrayWithContentsOfURL:error:)]) { //-- ios11
```

Comment:
Not sure if there is a better to way to fix this without checking OS version/detecting availability of API.

Reply:
there is a better way to validate the ios version (11), but is only available on xcode9. it would get broken for anyone (still using xcode 8)

if (@available(iOS 11.0, *)) {
    NSLog(@"newest");
} else {
    NSLog(@"not newest");
}